proposals/0002-explicit-fixity-(~).md

```diff
@@ -0,0 +1,55 @@
+---
+author: Rinat Stryungis
+date-accepted: ""
+ticket-url: "https://gitlab.haskell.org/ghc/ghc/-/issues/18252"
+implemented: "Yes"
+---
+
+This proposal is [discussed at this pull request](https://github.com/haskell-core/core-libraries-proposals/pull/0>).
+**After creating the pull request, edit this file again, update the number in
+the link, and delete this bold sentence.**
+
+# Proposal title
+
+Added explicit fixity for `(~)` and `(~~)`
```

Comment:
And here, according to the template, should be "a short abstract motivating and briefly summarizing the proposed change". So I'm thinking of something along the lines of:

```
Set the fixity of `(~)` and `(~~)` to 4, as it is consistent with the fixity of other equality operators, such as `(:~:)`, `(:~~:)`, and `(==)`.
```